src/helper.ts

```diff
@@ -0,0 +1,13 @@
+import { Photo } from "./photo";
+
+export function getHtml(photos: Photo[]): string {
+    let result = "<div>"
+    for (let i = 0; i < photos.length; i++){
+        const url: string = photos[i].src;
+        const id: number= photos[i].id;
+        const photoStr: string = `<div> <img src = ${url} width = "200" height = "200"> <p>Picture id: ${id}</p></div>`;
```

Comment:
My suspicion is that it's probably fine, but I'd expect there to be quotes around `${url}` in order to make it proper HTML. Can you give it a try for me? Even if it works both ways, it'd be better to have properly formed HTML instead of having the browser guess.